data_scraper.py

```diff
@@ -150,17 +174,26 @@ def _data_cleansing(self, df):
 class RKIDataCountyCsv(CsvDataSource):
     def __init__(self):
         super(RKIDataCountyCsv, self).__init__(
-                url="https://opendata.arcgis.com",
-                endpoint="datasets/917fc37a709542548cc3be077a786c17_0.csv",
-                info="Data from the Robert-Koch-Institut on the current cases per county.",
-            )
+            name="RKI_county_level",
+            url="https://opendata.arcgis.com",
+            endpoint="datasets/917fc37a709542548cc3be077a786c17_0.csv",
+            info="Data from the Robert-Koch-Institut on the current cases per county.",
+        )
 
 
 class RKIDataStateCsv(CsvDataSource):
     def __init__(self):
         super(RKIDataStateCsv, self).__init__(
+            name="RKI_state_level",
             url="https://opendata.arcgis.com",
             endpoint="datasets/ef4b445a53c1406892257fe63129a8ea_0.csv",
             info="Accumulated cases in federal states in Germany as per Robert-Koch-Institut.",
-            date_columns["Aktualisierung"]
+            date_columns=["Aktualisierung"]
         )
+
+
+def lambda_handler(message, context):
+    s3_client = boto3.client("s3")
+    hamburg_clinics = HamburgClinics()
+    response = hamburg_clinics.write_to_s3(s3_client)
+    return response
```

Comment:
Shouldn't this write *all* datasets to s3 instead of only the HamburgClinics one??